src/components/transactions/MigrateV3/MigrateV3ModalContent.tsx

```diff
@@ -68,7 +70,34 @@ export const MigrateV3ModalContent = () => {
   if (txError && txError.blocking) {
     return <TxErrorView txError={txError} />;
   }
-  if (migrateTxState.success) return <TxSuccessView action={<Trans>Migrated</Trans>} />;
+
+  const handleRoute = () => {
+    if (currentMarket === 'proto_polygon') {
+      setCurrentMarket('proto_polygon_v3' as CustomMarket);
+      window.location.href = '/?marketName=proto_polygon_v3';
```

Comment:
You could use the CustomMarket enum values directly here, might be a little cleaner. Something like:
```ts
if (currentMarket === CustomMarket.proto_polygon) {
       setCurrentMarket(CustomMarket.proto_polygon_v3);
       window.location.href = `/?marketName=${CustomMarket.proto_polygon_v3}`;
} else if ...
```

Reply:
good catch! added here `ed5bf838ccfb929a92fbc44baa5602c76e4ea670`